type=class
superclass=StandardError
included=
extended=
dynamically_included=
dynamically_extended=
library=_builtin
aliases=
aliasof=
source_location=refm/api/src/_builtin/RuntimeError:1

特定の例外クラスには該当しないエラーが起こったときに発生します。
また [[m:Kernel.#raise]] で例外クラスを指定しなかった場合も
RuntimeError が発生します。
